src/chrome/app/manifest.json

```diff
@@ -23,5 +23,9 @@
     "tcpServer": { "listen": "" },
     "udp": { "send": "", "bind": "" }
   },
+  "externally_connectable": {
+    "ids" : ["pjpcdnccaekokkkeheolmpkfifcbibnj"],
+    "matches" : ["*://*.uproxy.org/*", "*://1-0-1-test-dot-uproxysite.appspot.com/*"]
```

Comment:
Can you do "*://*-dot-uproxysite.appspot.com/*" so that this will continue to work with new test versions?